src/taco-cli/package.json

```diff
@@ -54,7 +54,7 @@
         "gulp": "^3.8.10",
         "gulp-concat": "^2.4.2",
         "mocha": "latest",
-        "should": "*"
+        "should": "4.3.0"
```

Comment:
we should create a bug/issue to track moving us to latest/7. This would help us leverage all the fixes and new features in should library